go.mod

```diff
@@ -66,3 +66,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/ailinkid/parser v0.0.0-20200727081704-30ae78e3ace2
```

Comment:
Fix this. This pr should be picked into release-4.0.

Reply:
ok